### PR TITLE
Minor enhancement

### PR DESCRIPTION
**Pull request by wsliaw**

### CODE REVIEW

File: iphone/Classes/TiBluetoothPeripheralManagerProxy.m [hunk 1]
```diff
@@ -159,7 +159,7 @@ - (void)updateValueForCharacteristicOnSubscribedCentrals:(id)args
         [result addObject:[(TiBluetoothCentralProxy *)subscribedCentral central]];
     }
     
-    [peripheralManager updateValue:[(TiBlob *)value data]
+    return [peripheralManager updateValue:[(TiBlob *)value data]
```
Comment (hansemannn) [inline iphone/Classes/TiBluetoothPeripheralManagerProxy.m]: Does this work for you? I would suspect you need to return an `NSNumber *` and wrap the return in `NUMBOOL()`.

Comment (wsliaw) [inline iphone/Classes/TiBluetoothPeripheralManagerProxy.m]: Yes it works for me, but I agree with you. Can you help me on this? not sure how to implement that

Comment (hansemannn) [inline iphone/Classes/TiBluetoothPeripheralManagerProxy.m]: Sure!

```objc
 - (NSNumber *)updateValueForCharacteristicOnSubscribedCentrals:(id)args
  {
    ENSURE_ARG_COUNT(args, 3);
      
     id value = [args objectAtIndex:0];
     id characteristic = [args objectAtIndex:1];
     id subscribedCentrals = [args objectAtIndex:2];
     
     ENSURE_TYPE(value, TiBlob);
     ENSURE_TYPE(characteristic, TiBluetoothCharacteristicProxy);
     ENSURE_TYPE(subscribedCentrals, NSArray);
     
     NSMutableArray *result = [NSMutableArray array];
     
     for (id subscribedCentral in subscribedCentrals) {
         ENSURE_TYPE(subscribedCentral, TiBluetoothCentralProxy)
          [result addObject:[(TiBluetoothCentralProxy *)subscribedCentral central]];
      }
      
      return NUMBOOL([peripheralManager updateValue:[(TiBlob *)value data] forCharacteristic:(CBMutableCharacteristic *)[(TiBluetoothCharacteristicProxy *)characteristic characteristic]
                onSubscribedCentrals:result]);
  }
```

**Comment by hansemannn** [comment]
LGTM, approved! Thank you!

**Comment by hansemannn** [comment]
New release: https://github.com/hansemannn/titanium-bluetooth/releases/tag/ios-v1.2.1

**Pull request merged**